Classes/Transfer/Exception.php

```diff
@@ -10,35 +12,27 @@
  * information, please view the LICENSE file which was distributed with this
  * source code.
  */
+
 use Flowpack\ElasticSearch\Exception as ElasticSearchException;
-use Neos\Flow\Http\Request;
-use Neos\Flow\Http\Response as FlowResponse;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
 
 /**
  * Exception that occurs related to ElasticSearch transfers
  */
 class Exception extends ElasticSearchException
 {
     /**
-     * @var FlowResponse
+     * @var ResponseInterface
      */
     protected $response;
 
     /**
-     * @var Request
+     * @var RequestInterface
      */
     protected $request;
 
-    /**
-     * Exception constructor.
-     *
-     * @param string $message
-     * @param int $code
-     * @param FlowResponse $response
-     * @param Request $request
-     * @param \Exception $previous
-     */
-    public function __construct($message, $code, FlowResponse $response, Request $request = null, \Exception $previous = null)
+    public function __construct($message, $code, ResponseInterface $response, RequestInterface $request = null, \Exception $previous = null)
```

Comment:
Not sure about the dropped docblock – two arguments are now without type hint and in theory we still require docblocks in our CGL.